src/main/java/org/ow2/proactive/catalog/report/CatalogObjectReportPDFGenerator.java

```diff
@@ -80,62 +100,150 @@
         }
     }
 
-    private BaseTable createAndIntitializeBaseTable(PDDocument doc, PDPage page) throws IOException {
-        //Initialize table
-        float margin = 10;
+    private void populateTable(List<String[]> catalogObjects, BaseTable table)
+            throws MalformedURLException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.net.MalformedURLException' which is a subclass of 'java.io.IOException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)